### ISSUE TITLE
Deprecated CsrfProviderAdapter still present

### ISSUE THREAD

**Issue by zanardigit** [symfony/symfony#15093]
In my project with Symfony 2.7.1 I get this "deprecated" warnings:

DEPRECATED - The Symfony\Component\Form\Extension\Csrf\CsrfProvider\CsrfProviderAdapter class is deprecated since version 2.4 and will be removed in version 3.0. Use the Symfony\Component\Security\Csrf\CsrfTokenManager class instead.

But I see this class is still used in several other Symfony classes:

* src/Symfony/Component/Form/Extension/Csrf/EventListener/CsrfValidationListener.php
* src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php
* src/Symfony/Component/Form/Extension/Csrf/CsrfExtension.php
* src/Symfony/Component/Form/Extension/Templating/TemplatingExtension.php
* src/Symfony/Component/Form/FormRenderer.php
* src/Symfony/Component/Security/Http/Logout/LogoutUrlGenerator.php
* src/Symfony/Component/Security/Http/Firewall/LogoutListener.php
* src/Symfony/Component/Security/Http/Firewall/SimpleFormAuthenticationListener.php
* src/Symfony/Component/Security/Http/Firewall/UsernamePasswordFormAuthenticationListener.php

So basically there is no way for me to fix this "deprecated" warning. Is that expected?

**Comment by xabbuh** [comment]
@zanardigit Maybe you are using the `form.csrf_provider` service instead of the `security.csrf.token_manager`.

**Comment by zanardigit** [comment]
@xabbuh you're correct. Thank you!
